pp_netlib/network.py

```diff
@@ -235,10 +236,58 @@ def get_summary(self, print_to_std = True, summary_file_prefix = None):
                 summary.write(summary_contents)
             summary.close()
 
-    def visualize(self):
-        # code for calling viz functions
-        print("visualizing network")
-        return #files associated with viz
+    def visualize(self, viz_type, out_prefix, external_data = None):
```

Comment:
Are there any restrictions on outprefix? i.e. can it be both an absolute and relative path?

Reply:
the outprefix is a filename prefix that is applied to all output files, rather than a path. the output path is set to self.outdir/mst or self.outdir/cytoscape as the case may be